app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError
```

Comment:
The `ValueError` should include a message to indicate what went wrong. Consider adding a message like `raise ValueError('Invalid command format')`.